.github/actions/pylint/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "pylint-oaio-gh-action",
+    "version": "1.0.0",
+    "description": "Run pylint as a GitHub action",
+    "main": "index.js",
+    "scripts": {
+        "build": "ncc build index.js --license licenses.txt"
```

Comment:
[fix] I think we need a `README.md` that explains the development process here. Alternately, if we hosted this in a separate `REPO` is there an off-the-shelf GitHub workflow that will build/release actions based on tags?

Reply:
I'll address the issues in this repo first, then when it's looking OK we can abandon it and move the code to a new repo. Sound sensible?

Reply:
A fine plan indeed :)

Reply:
Added a README in a454ff8 based on the template in [the docs](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#creating-a-readme)

Reply:
Updated the README to actually address your comment in 2f51b2f :roll_eyes: